api/db/seeds/data/team-acces/build-oidc-providers.js

```diff
@@ -0,0 +1,20 @@
+import Debug from 'debug';
+
+const debugOidcProvidersSeeds = Debug('pix:oidc-providers:seeds');
+
+export async function buildOidcProviders(databaseBuilder) {
+  const oidcProvidersJson = process.env.OIDC_PROVIDERS;
+  if (!oidcProvidersJson) {
+    debugOidcProvidersSeeds('No environment variable OIDC_PROVIDERS defined, no loading of "oidc-providers" table.');
+    return;
+  }
+
+  const oidcProviders = JSON.parse(oidcProvidersJson);
+  debugOidcProvidersSeeds({ oidcProviders });
```

Comment:
**question**
Est-ce utile de logger l'ensemble du fichier dans la console ? Même si ce n'est qu'en debug, ça ne me paraît pas une bonne pratique d'afficher des clés privés.

On pourrait remplacer par un log d'info (non debug donc)  avec juste une ligne `Loaded configuration for Identity provider {identityProvider}’` pour chacun ?